test-pad/test-pad

```diff
@@ -253,25 +261,28 @@ elif  [ "${REPO}" = "k3s" ]; then
     STORAGE_URL="https://storage.googleapis.com/k3s-ci-builds"
     BINARY="k3s"
     # If not assigned, use alpine-3.16 for fast download and startup time
-    OS=${OS:-generic/alpine316}
+    OS=generic/alpine316
 elif [ "${REPO}" = "rke2" ]; then
     ORG="rancher"
     BINARY="rke2.linux-amd64"
     STORAGE_URL="https://storage.googleapis.com/rke2-ci-builds"
     # RKE2 requires systemd, so we use ubuntu,
     # note that it is much larger (~1.5GB) and takes longer to startup.
-    OS=${OS:-generic/ubuntu2004}
+    OS=generic/ubuntu2004
 else 
    echo "error: unknown repo (-r) $REPO"
    exit 1
 fi
 
-if [ "${OS}" = "alpine" ]; then 
-    OS=generic/alpine316; 
-elif [ "${OS}" = "ubuntu" ]; then 
+if [ "${OSFLAG}" = "alpine" ]; then 
```

Comment:
If this still contains the flag, the variable name of OS is more descriptive unless this contains a flag and then the rename would be accurate.

Reply:
I am trying to make it easier on the user so they don't have to type the full OS name and allows OS upgrades behind-the-scenes. 

So OSFLAG is the flag passed by the user: `alpine` `ubuntu` `debian`.
The OS is the actual true name of the vagrant box: `generic/alpine316`, `generic/ubuntu2004`, `generic/debian11`

Do you have a suggestion on better naming?

Reply:
Nah, that works. I must have missed that nuance. :)